docs/options.md

```diff
@@ -193,3 +193,25 @@ By default, Prettier will wrap markdown text at the specified print width. In so
 | Default | CLI Override      | API Override        |
 | ------- | ----------------- | ------------------- |
 | `true`  | `--no-prose-wrap` | `proseWrap: <bool>` |
+
+## Indent Styles
```

Comment:
I think we need some discussion about the option naming, it's really confusing me what does `--indent-styles` mean at first glance, I personally would recommend something like `--css-hierarchy-indent`.

And there should be an `_available in v1.9.0+_` or something indicating that's an unreleased option.

Reply:
Good point. I'll do that. I certainly agree that `indent-styles` is a bit unnecessarily confusing. I chose "styles" to encompass Sass/Less/CSS but "css" is probably simpler and clearer language. I've just searched through the various links posted in this PR/Issue and "indenting" is the most popular term amongst the community, with "hierarchy" being a close second (although it isn't mentioned in [guides like this](https://cssguidelin.es/#indenting)), so your suggestion makes sense to me.